README.md

```diff
@@ -144,6 +144,22 @@ const { getWilayaByPhoneCode } = require('@dzcode-io/leblad');
 console.log(getWilayaByPhoneCode(34)); //will the wilaya object ({name: "Béjaïa"...})
 ```
 
+#### getPhoneCodesForWilaya(wilayaCode?: number)
```

Comment:
This field is required

```suggestion
#### getPhoneCodesForWilaya(wilayaCode: number)
```

Reply:
Should be fixed! I noticed a '?' on a few of the other methods as well. Did you want me to clean those up as well?